### PR TITLE
doc: update the moderation policy

### PR DESCRIPTION
**Pull request by jasnell**
Fill in some ambiguous gaps in the moderation policy

/cc @nodejs/tsc

### CODE REVIEW

File: Moderation-Policy.md [hunk 1]
```diff
@@ -31,10 +31,11 @@ Any alternative Moderation Policy used for a given repository must be included i
 
 ### Terms
 
-* *Collaborator* refers to any individual with configured access permissions to any Node.js GitHub organization repository. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
+* *Collaborator* refers to any individual with configured write (commit) permissions to any Node.js GitHub organization repository *other than the Moderation Repository*. See [GitHub's access permissions documentation](https://help.github.com/articles/what-are-the-different-access-permissions/) for more information.
```
Comment (nebrius) [inline Moderation-Policy.md]: Does everyone in node/membership have write access specifically to the moderation repo? I could be wrong, but I thought I remembered that only a subset of people have write access.

Comment (jasnell) [inline Moderation-Policy.md]: Every Collaborator should have at least access to view the Moderation Repository and to participate in discussions.

**Comment by Charlotteis** [comment]
This certainly clarifies a few things for me. Thanks jasnell. I particularly like the `write (commit)` addition to the Collaborator definition as this is a really good way to describe it.

**Comment by bnoordhuis** [comment]
LGTM

File: Moderation-Policy.md (hunk 1 above)
Comment (nebrius) [inline Moderation-Policy.md]: I could be wrong, but I thought that being a member of the org gave read access, but not write.

On further reflection though, if this is the case, it doesn't make the wording here wrong. Absolute worst case it's a bit superfluous, and best case it makes an important distinction, so there's no downside. I'm ok with this.

**Comment by nebrius** [comment]
LGTM

**Comment by mhdawson** [comment]
LGTM

**Comment by TheAlphaNerd** [comment]
@jasnell can this land?

**Comment by jasnell** [comment]
Yep

**Comment by williamkapke** [comment]
Lotsa LGTMs here. Merging.

**Pull request merged**